statistics/handle/cache/internal/lfu/lfu_cache.go

```diff
@@ -32,6 +34,14 @@ type LFU struct {
 
 // NewLFU creates a new LFU cache.
 func NewLFU(totalMemCost int64) (*LFU, error) {
+	if totalMemCost == 0 {
+		memTotal, err := memory.MemTotal()
+		if err != nil {
+			return nil, err
+		}
+		totalMemCost = int64(memTotal / 2)
```

Comment:
Looks like this is a behavior change, is there any rationale behind this change?

Reply:
In the old LRU, If you set capacity 0, you will have unlimited memory in the cache. It is dangerous.